src/components/modalWindow/index.js

```diff
@@ -46,9 +52,49 @@ export default class ModalWindow {
 
         yesButton.render();
         noButton.render();
+        document.querySelector(".modalWindow__button-close").addEventListener("click", () => this.closeWithX());
```

Comment:
`.addEventListener("click", () => this.closeWithX());` -> `.addEventListener("click", this.closeWithX);`